src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -31,14 +31,15 @@ import { ext } from '../../../extensionGlobals'
 export interface AddSamDebugConfigurationInput {
     resourceName: string
     rootUri: vscode.Uri
+    apiEvents?: Map<string, CloudFormation.Event>
```

Comment:
I'm not sure why you're using a map here? If I'm reading the above correctly, you're only setting a single entry per event?

Reply:
probably enough to rename it to `apiEvent` (singular), or I guess have 2 fields `apiEventName`, `apiEvent`